ucsmsdk/ucssession.py

```diff
@@ -549,21 +549,19 @@ def _logout(self):
         if self.__refresh_timer:
             self.__refresh_timer.cancel()
 
-        if self.__cookie:
```

Comment:
@ragupta-git we already have `if self.__cookie is None` at the beginning of `_logout` and we return if cookie is not valid. So, not sure why this was required.